<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Optimize</class>
 <widget class="QDialog" name="Optimize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLIS - Optimize</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QCustomPlot" name="plot_area" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>222</width>
        <height>412</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="2">
        <widget class="QCheckBox" name="kd2_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="kd2">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="intmed_label">
         <property name="text">
          <string>&lt;I&gt;s&lt;/I&gt;(PL):</string>
         </property>
        </widget>
       </item>
       <item row="24" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="start_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QDoubleSpinBox" name="end2">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-4000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QLineEdit" name="chi2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="kdc_label">
         <property name="text">
          <string>Kdc:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QDoubleSpinBox" name="end">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-4000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4000000.990000000223517</double>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QCheckBox" name="end_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="site_label">
         <property name="font">
          <font>
           <family>Helvetica</family>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Site:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="kd1_label">
         <property name="text">
          <string>Kd1:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="kd1">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="25" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Optimize</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="kdc">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="kd1_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Helvetica</family>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Fixed</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="start">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-4000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="kd2_label">
         <property name="text">
          <string>Kd2:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="end_label">
         <property name="text">
          <string>&lt;I&gt;s&lt;/I&gt;(PL&lt;sub&gt;2&lt;/sub&gt;):</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QDoubleSpinBox" name="intmed1">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-4000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4000000.990000000223517</double>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Chi2</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="end_label2">
         <property name="text">
          <string>&lt;I&gt;s&lt;/I&gt;(PL&lt;sub&gt;3&lt;/sub&gt;):</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QCheckBox" name="intmed_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="kd4_label">
         <property name="text">
          <string>Kd3:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="kd4">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="start_label">
         <property name="text">
          <string>&lt;I&gt;s&lt;/I&gt;(P):</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="Kdc_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="kd4_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="end_label3">
         <property name="text">
          <string>&lt;I&gt;s&lt;/I&gt;(PL&lt;sub&gt;3&lt;/sub&gt;):</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QDoubleSpinBox" name="end3">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-4000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>4000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QCheckBox" name="end2_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QCheckBox" name="end3_check">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Optimize the curve</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
